sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/package-info.java

```diff
@@ -2,8 +2,453 @@
 // Licensed under the MIT License.
 
 /**
- * Package containing classes for creating {@link com.azure.search.documents.SearchClient} or
- * {@link com.azure.search.documents.SearchAsyncClient} used to perform document management, autocomplete, search, or
- * suggestion operations using an Azure Cognitive Search service index.
+ * <p><a href="https://learn.microsoft.com/azure/search/">Azure AI Search</a>, formerly known as "Azure Cognitive Search, provides secure information retrieval at scale over
+ * user-owned content in traditional and conversational search applications.</p>
+ *
+ * <p>The Azure AI Search service provides:/p>
```

Comment:
Should this entire package-info Javadoc be copied to the /indexes/package-info.java Javadoc as well? Or should portions of this Javadoc be split out into that? As the top-level package is for the documents-based clients and /indexes is for the index creation and indexing clients.

Reply:
@alzimmermsft I can see merits to both options and so I'm not leaning particularly in any direction. Do you favor one approach over the other?

Reply:
I'm good with both having all for now